commandapi-platforms/commandapi-velocity/commandapi-velocity-core/src/main/java/dev/jorel/commandapi/CommandAPIVelocityConfig.java

```diff
@@ -15,10 +19,41 @@ public class CommandAPIVelocityConfig extends CommandAPIConfig<CommandAPIVelocit
 	 * @param server The {@link ProxyServer} that the CommandAPI is running on.
 	 */
 	public CommandAPIVelocityConfig(ProxyServer server) {
+		this(server, null);
+	}
+
+	/**
+	 * Creates a new CommandAPIVelocityConfig object. Variables in this
+	 * constructor are required to load the CommandAPI on Velocity properly.
+	 *
+	 * @param server The {@link ProxyServer} that the CommandAPI is running on.
+	 * @param plugin The plugin that is loading the CommandAPI.
+	 */
+	public CommandAPIVelocityConfig(ProxyServer server, Object plugin) {
 		this.server = server;
+		this.plugin = plugin;
 		super.setNamespace("");
 	}
```

Comment:
Thought about it. Decided not to go with it because it isn't backwards compatible.
I guess it doesn't matter though since technically Velocity only exists as snapshot releases until 10.0.0.
Do you think it's viable to delete the constructor with only one parameter?

Reply:
Yeah, I feel like it's fine to make breaking changes in snapshots. But, we could also leave it like it is here and make that break when merging #467.

Reply:
Well, I did it. I thought about it the last few hours but realized we might as well just do it now since it's going to change anyway.